Data/models/Bookmark.swift

```diff
@@ -507,10 +507,10 @@ extension Bookmark {
     func updateResolvedRecord(_ record: SyncRecord?, context: WriteContext = .new) {
         guard let bookmark = record as? SyncBookmark, let site = bookmark.site else { return }
         title = site.title
+        syncParentUUID = bookmark.parentFolderObjectId
```

Comment:
How does this affect code? I am sorry if I've missed something. trying to figure out

Reply:
I just moved it up few lines, previously it was set after the `updateInternal` and it was taking wrong parent Bookmark